frontend_tests/node_tests/i18n.js

```diff
@@ -7,7 +7,7 @@ zrequire("templates");
 // that here for the tests.
 set_global("page_params", {
     translation_data: {
-        "Quote and reply": "French translation",
+        "Quote and reply / forward": "French translation",
```

Comment:
maybe "quote and reply or forward" reads better?

Reply:
updated! Sorry, somehow forgot to update this.